hooks/templatedatavalidator.js

```diff
@@ -0,0 +1,115 @@
+var fs = require('fs');
+var path = require('path');
+const { parseJamboConfig } = require('../commands/helpers/utils/jamboconfigutils');
+const { error } = require('../commands/helpers/utils/logger');
+/**
+ * Validates page template's data configuration
+ * 
+ * @param {Object} pageData configuration of a page template
+ * @returns {boolean} false if validator should throw an error
+ */
+module.exports = function (pageData) {
+  const jamboConfig = parseJamboConfig();
+  return isGlobalConfigValid(pageData.global_config) && isPageVerticalConfigValid(pageData, jamboConfig);
```

Comment:
maybe we can also have the non short circuiting logic here too, so we call `isPageVerticalConfigValid` even if the global config is not valid?